### PR TITLE
Arkad white belt promotion

### PR DESCRIPTION
**Pull request by lontivero**
I would like to promote **Arkad** to Wasabi Dojo as a White Belt member for his efforts in the creation of educational material about privacy in general and Wasabi in particular with his latest Wasabi Wallet tutorial for beginners and intermediate/advanced users.

Arkad also shares our focus on privacy and educates bitcoiners about this using different tools such as ColdCard, Electrum Wallet, Wasabi Wallet, Samourai Wallet and Bitcoin Core among others. He also educates on many technical aspects and write articles.    

Update: Arkad's target is the Spanish speaking community.

### CODE REVIEW

File: docs/building-wasabi/Dojo.md [hunk 1]
```diff
@@ -67,10 +67,10 @@ Osu!
...
 | Dennis Reimann   | [Twitter][47]               | [Wasabi Documentation][48]   | 2019-08: white belt by Max Hillebrand|
 | ThunderBiscuit   | [Twitter][46]               | [Wasabi Documentation][49]   | 2019-08: white belt by Max Hillebrand|
-| Riccardo Masutti   | [Twitter][50]               | [Wasabi Documentation][51]   | 2019-11: white belt by Max Hillebrand|
-| Rafael Yakobi   | [Twitter][52]               | [Wasabi Guide][53]   | 2020-03: white belt by Max Hillebrand|
-| Greg Hajdu   | [Twitter][54]              | [Naming of the new CoinJoin technology][56] - [Wabi Sabi][55]   | 2020-04: white belt by David Molnar|
-
+| Riccardo Masutti | [Twitter][50]               | [Wasabi Documentation][51]   | 2019-11: white belt by Max Hillebrand|
+| Rafael Yakobi    | [Twitter][52]               | [Wasabi Guide][53]   | 2020-03: white belt by Max Hillebrand|
+| Greg Hajdu       | [Twitter][54]               | [Naming of the new CoinJoin technology][56] - [Wabi Sabi][55]   | 2020-04: white belt by David Molnar|
+| Arkad            | [Twitter][57]               | [Multiple video tutorial I][58] and [II][59]| 2020-04: white belt ny lontivero |
```
Comment (yahiheb) [inline docs/building-wasabi/Dojo.md:73]:
```suggestion
| Arkad            | [Twitter][57]               | Multiple video tutorials [I][58] and [II][59]| 2020-04: white belt by lontivero |
```

File: docs/building-wasabi/Dojo.md [hunk 2]
```diff
@@ -126,6 +126,10 @@ Osu!
 [54]: https://twitter.com/gergely_hajdu
 [55]: https://imgur.com/HpEMnIm
 [56]: https://github.com/zkSNACKs/WabiSabi
+[57]: https://twitter.com/Multicripto
+[58]: https://twitter.com/lunaticoin/status/1250737978906943488?s=20
+[59]: https://twitter.com/Multicripto/status/1250505298382643201?s=20
+
```
Comment (yahiheb) [inline docs/building-wasabi/Dojo.md:132]:
```suggestion
```

File: docs/building-wasabi/Dojo.md [hunk 3]
```diff
@@ -126,6 +126,10 @@ Osu!
 [54]: https://twitter.com/gergely_hajdu
 [55]: https://imgur.com/HpEMnIm
 [56]: https://github.com/zkSNACKs/WabiSabi
+[57]: https://twitter.com/Multicripto
+[58]: https://twitter.com/lunaticoin/status/1250737978906943488?s=20
+[59]: https://twitter.com/Multicripto/status/1250505298382643201?s=20
```
Comment (yahiheb) [inline docs/building-wasabi/Dojo.md:131]: Maybe it is better to remove `?s=20`?

File: docs/building-wasabi/Dojo.md (hunk 2 above)
Comment (lontivero) [reply docs/building-wasabi/Dojo.md:132]: This space has no effect.

**Pull request merged**